From e2fa564c8e7133a5a7d615d5f1e5c3bc72c60d2a Mon Sep 17 00:00:00 2001
From: Artem Bityutskiy <Artem.Bityutskiy@nokia.com>
Date: Fri, 1 Feb 2008 13:48:49 +0200
Subject: [PATCH] UBI: do not flush queue on each vtbl change

This is just not necessary. We re-write whole layout copy, so
the old contents cannot show up again sice scan process will
drop it.

Signed-off-by: Artem Bityutskiy <Artem.Bityutskiy@nokia.com>
---
 drivers/mtd/ubi/vtbl.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/mtd/ubi/vtbl.c b/drivers/mtd/ubi/vtbl.c
index d8222db..56fc3fb 100644
--- a/drivers/mtd/ubi/vtbl.c
+++ b/drivers/mtd/ubi/vtbl.c
@@ -111,7 +111,7 @@ int ubi_change_vtbl_record(struct ubi_device *ubi, int idx,
 	}
 
 	paranoid_vtbl_check(ubi);
-	return ubi_wl_flush(ubi);
+	return 0;
 }
 
 /**
-- 
1.5.6.3

